homeassistant/components/mysensors/gateway.py

```diff
@@ -281,17 +281,17 @@ def _validate_child(gateway, node_id, child):
     if not child.values:
         _LOGGER.debug(
             "No child values for node %s child %s", node_id, child.id)
-        return validated
+        return dict(validated)
```

Comment:
Why would this method make a copy when it returns? Seems to me that the caller of this method, who intents to mutate the value, should make a copy.